src/dialogs/device-registry-detail/dialog-device-registry-detail.ts

```diff
@@ -90,7 +93,12 @@ class DialogDeviceRegistryDetail extends LitElement {
               .disabled=${this._submitting}
             ></paper-input>
             <div class="area">
-              <paper-dropdown-menu label="Area" class="flex">
+              <paper-dropdown-menu
+                label="${this.hass.localize(
+                  "ui.panel.config.integrations.config_entry.area_picker_label"
```

Comment:
```suggestion
                  "ui.panel.config.devices.area_picker_label"
```

Reply:
Implemented in https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/96f6a843a110bbac601076e246129eb7376adc0b